Orange/canvas/report/report.py

```diff
@@ -256,7 +256,8 @@ def report_list(data,
         else:
             body = None
         if body:
-            self.report_html += "<table>\n" + body + "</table>"
+            self.report_html += "<table border = '1px' style = 'border-collapse:collapse'>\n" + \
```

Comment:
HTML attributes and their values are, by convention, [not separated with spaces](http://www.w3schools.com/html/html_attributes.asp).